tests/agent/nodejs/Makefile

```diff
@@ -6,4 +6,4 @@ stop:
 	docker-compose down -v
 
 setup:
-	npm install elastic-apm --save
+	npm install elastic/apm-agent-nodejs --save
```

Comment:
For now I would keep the latest from master as this is for manual testing and normally what I want to do is testing the most recent changes of the node agent to see how they work with the apm-server.